dist/sgx/README-cli.md

```diff
@@ -6,12 +6,267 @@ The computer on which the powHSM setup and onboarding is to be executed needs th
+Attempting to validate self-signed root authority...
+Using https://certificates.trustedservices.intel.com/Intel_SGX_Provisioning_Certification_RootCA.pem as root authority
+--------------------------------------------------------------------------------------------
+powHSM verified with public keys:
+m/44'/0'/0'/0/0:   025fd1ea7ff3b0fea07082942c203da7a7e69d7de53421621139e12ad93c563c9c
+m/44'/1'/0'/0/0:   02e4d3fd364d0b1f9cffd6cfd736f3491de343df6f634ebc1f70dff175b700be63
+m/44'/1'/1'/0/0:   029d1a5ea03384a35f735d3390de77ab2153d29d14f47b231d65a0d653b4417170
+m/44'/1'/2'/0/0:   0397650fb72878e6651a23c384638c25ac8e21052d23c70dd6eb64db8c3234b546
+m/44'/137'/0'/0/0: 02d6a13573ad391715751a12e83b391dbf7a237dc5abeb4b708382f106ae40a6f2
+m/44'/137'/1'/0/0: 020b792254ba4a31d9bd5eafe8ccef0c53119b248b8f383d41af19619193ce785e
+Hash: c010057e43ffd4c1db683b03511c6e06f8b8a73f2d7e8a4fce10f2728707c851
+
+Installed powHSM MRENCLAVE: 72116efdbdf3fdf415eebbded0be15f6e81149ef7adcca36a0108e83b7ac7cb2
+Installed powHSM MRSIGNER: dcba724ceb3ce4c93aece10c935eda46d652856dc0c6bf634a8a7961c0e1db68
+Installed powHSM version: 5.4
+Platform: sgx
+UD value: 116dfb22c3bfc58404ded444fac16b7961e6c68b129692b872ac68b31e2c4557
+Best block: bdab1b4b4a768d1e32ce538a8121022d93059e467fa834e090db1f2c0db00523
+Last transaction signed: 0000000000000000
+Timestamp: 0
+--------------------------------------------------------------------------------------------
+
+Stopping the powHSM...
+
+powHSM Setup complete. Find the installation in /opt/powHSM.
+```
+
+Once that step is finished, all the files required by the powHSM will be located in the installation directory:
+```
+/opt/powHSM/
+├── bin
+│   ├── Dockerfile
+│   ├── hsmsgx
+│   ├── hsmsgx_enclave.signed
+│   ├── start
+│   └── stop
+├── kvstore-nvmem-bcstate.dat
+├── kvstore-nvmem-bcstate_updating.dat
+├── kvstore-password.dat
+├── kvstore-retries.dat
+└── kvstore-seed.dat
+```
+
+### powHSM service setup
+Once installation is complete, the script will proceed with the setup of the powHSM service. The powHSM application
+is installed as a `systemd` service. The last piece of information required by the script is the name of the docker
+network to which the container running the powHSM will be connected. The default value `net_sgx` is suitable for most
+cases, but the user is free to choose a different name. Just pressing `Enter` will use the default value:
+```
+Creating hsm user and group...
+Enter the name of the docker network to be created: [net_sgx]
+>
+```
+
+The user will be required to confirm the creation of the docker network:
+```
+The docker network will be named 'net_sgx'. Proceed? [Y/n]
+> Y
+```
+
+After confirming the creation of the docker network, the script will proceed with the setup of the powHSM service. All
+the following steps are automated and require no user intervention:
+```
+Creating net_sgx network...
+Setting permisions...
+Creating service...
+Enabling service...
+EStarting service...
```

Comment:
There's an extra `E` at the beginning of this line.

Reply:
Nice catch, this is actually a typo in the script. I'm fixing both